frontend/src/views/EventMaking/EventSettingPage.vue

```diff
@@ -1,99 +1,145 @@
-                <option v-for="sports_discipline in sports_disciplines" :key="sports_discipline.id" :value="sports_discipline">
-                  {{ sports_discipline.name }}
-                </option>
-              </select>
+            <div class="error">{{ sports_type_selected_error }}</div>
+            <li class="md:grid md:grid-cols-12 md:gap-4 md:items-center">
+              <label class="md:col-span-4 text-left px-3 py-2" for="discipline" v-if="sports_disciplines.length > 0">種目</label>
+              <div class="md:col-span-8">
+                <select class="w-full py-2 px-3 border-2 border-gray-200 box-border" id="discipline"  v-model="sports_discipline_selected" multiple v-if="sports_disciplines.length > 0">
+                  <option v-for="sports_discipline in sports_disciplines" :key="sports_discipline.id" :value="sports_discipline">
+                    {{ sports_discipline.name }}
+                  </option>
+                </select>
+              </div>
             </li>
             <div class="mb-5 mx-5 bg-white rounded-md">{{ sports_discipline_selected.length ? sports_discipline_selected.map(sport => sport.name).join(", ") : '' }}</div>
-            <li class="md:grid grid-cols-3 gap-4">
-              <label class="md:place-self-start md:mt-3 mx-4" for="prefecture">都道府県</label>
-              <select class="md:place-self-center mx-10 py-1 px-1 my-2 w-72 border-2 border-gray-200 box-border" id="prefecture"  v-model="prefecture_selected">
-                <option disabled value="">1つを選択して下さい</option>
-                <option v-for="prefecture in prefectures" :key="prefecture.id" :value="prefecture.id">
-                  {{ prefecture.name }}
-                </option>
-              </select>
+            <div class="md:-mb-16">
+              <li class="md:grid md:grid-cols-12 md:gap-4 md:items-center">
+                <label class="md:col-span-4 text-left px-3 py-2" for="prefecture">都道府県</label>
+                <div class="md:col-span-8">
+                  <select class="w-full py-2 px-3 border-2 border-gray-200 box-border" id="prefecture" v-model="prefecture_selected">
+                    <option disabled value="">1つを選択して下さい</option>
+                    <option v-for="prefecture in prefectures" :key="prefecture.id" :value="prefecture.id">
+                      {{ prefecture.name }}
+                    </option>
+                  </select>
+                </div>
+              </li>
               <div class="error">{{ prefecture_selected_error }}</div>
+            </div>
+            <li class="md:grid md:grid-cols-12 md:gap-4 md:items-center">
+              <label class="md:col-span-4 text-left px-3 py-2" for="event_url">イベントURL</label>
+              <div class="md:col-span-8">
+                <input class="w-full py-2 px-3 border-2 border-gray-200 box-border" id="event_url" type="url" placeholder="http://" v-model="event_url">
```

Comment:
placeholderはより具体的なurlを示した方が良いですが修正は不要です！

```suggestion
                <input class="w-full py-2 px-3 border-2 border-gray-200 box-border" id="event_url" type="url" placeholder="http://sample.com" v-model="event_url">
```

Reply:
修正しました。